### ISSUE TITLE
kubevirt-tutorial-presubmit-lab-testing-k8s

### ISSUE THREAD

**Issue by ptrnull** [kubevirt/kubevirt-tutorial#109]
Some of the python code written in the kubevirt-tutorial-presubmit-lab-testing-k8s is still in 2.7, and the following warning is appearing:

DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. 

That code has to be ported to python 3.

/cc jparrill

**Comment by jparrill** [comment]
There is not python code on the repo ://, maybe Ansible is using python2 on the testing stage instead of python3. Could you please share the logs where you've found that?

**Comment by ptrnull** [comment]
Yes, here they are: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt-tutorial/108/kubevirt-tutorial-presubmit-lab-testing-k8s-1.13.3/1158295193493966848

Line 31:
> hack/tests: line 68: type: ansible: not found
> DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
> Collecting kubernetes

**Comment by iranzo** [comment]
As discussed with Juanma, PR 111 current status is back to initial (but using python3), so it seems that when ansible connects, it seems to use python2 instead o python3 and hence, it fails
